cfme/roles.py

```diff
@@ -300,8 +300,9 @@ def _remove_from_all(roles, r_page):
                            'Replicators', 'Projects', 'Topology'],
             'Infrastructure': ['Datastores', 'Networking', 'Providers', 'Virtual Machines', 'Hosts',
                                'Clusters', 'Topology', 'PXE', 'Resource Pools'],
-            'Physical Infrastructure': ['Providers', 'Racks', 'Switches', 'Servers', 'Topology'],
-            'Migration': None},
+            'Physical Infrastructure': ['Overview', 'Providers', 'Chassis', 'Racks', 'Switches',
+                                        'Servers', 'Storages', 'Topology'],
```

Comment:
Q: Does the order matter here? I noticed that `Switches` comes after `Storages` but before `Topology` in CFME.

Reply:
The test that uses this is using DeepDiff with 'ignore_order=True', so the order does not matter. I tried however to add the items as they are in CFME UI, just for looks.

Reply:
@apagac great minds think alike, this was my exact thought process when creating this.